addons/blocks2image/addon.json

```diff
@@ -17,5 +17,10 @@
     }
   ],
   "versionAdded": "1.13.0",
-  "enabledByDefault": false
+  "enabledByDefault": false,
+  "latestUpdate": {
+    "version": "1.36.0",
+    "temporaryNotice": "You can now copy directly to the system clipboard."
+  },
+  "permissions": ["clipboardWrite"]
```

Comment:
In the future it could interesting to offer per-setting permissions and put the clipboard options behind a toggle.